source/include/FreeRTOS_DNS.h

```diff
@@ -59,6 +59,12 @@ extern const MACAddress_t xMDNS_MACAddressIPv6;
 /* The MDNS IPv6 address is ff02::1:3 */
 extern const IPv6_Address_t ipMDNS_IP_ADDR_IPv6;
 
+/* This type-name was formally "misspelled" as
+ * xMDNS_MACAddressIPv6 with "MAC": */
+extern const MACAddress_t xMDNS_MacAddressIPv6;
+/* Guarantee backward compatibility. */
+#define xMDNS_MACAddressIPv6    xMDNS_MacAddressIPv6
+
```

Comment:
Two people proposing the same change?
Please have a look at my [PR 1169](https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/pull/1169).
Thanks

Reply:
Hi @htibosch,
I just merged this from your #1169 to fix merge conflict.